ingests/dummy.meta.yml

```diff
@@ -0,0 +1,17 @@
+namespace: dummy
+short_name: dummy
+name: Dummy dataset
+source_name: dummy source
+publication_year: 2020
+publication_date: 
+url: https://www.dummy.com/
+source_data_url: https://www.rug.nl/ggdc/historicaldevelopment/maddison/data/mpd2020.xlsx
```

Comment:
* Should we include other fields, like date_accessed or version? If so, I think it would be better to have version as a date, instead of a year.
* Should we use another dummy dataset, instead of the GDP one?

Reply:
Added `date_accessed` and switched from `publication_date` to `publication_year`. Honestly, I find it all a bit confusing... maybe having a single field `version` with recommendations to use publication date would be the easiest. Having implicit `version` is weird.

```
@property
def version(self) -> str:
    return self.publication_year or self.publication_date
```

> Should we use another dummy dataset, instead of the GDP one?

Can you think of a better dataset? I personally use it for prototyping all the time.

Reply:
`date_accessed` is actually automatically set to today when creating `dummy.json` index file.